src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -14,7 +14,7 @@
 
 public class SortCommandTest {
     private Model model = new ModelManager(getUnsortedTypicalAddressBook(), new UserPrefs());
```

Comment:
for this as well, perhaps could change the instances of getUnsortedTypicalAddressBook() to fit our context